PULL_REQUEST_BAZEL_TARGETS

```diff
@@ -57,3 +56,23 @@
 
 # XNet compatibility test for changes to certifications
 *canonical_state/* //rs/tests/message_routing/xnet:xnet_compatibility
+
+*nns_delegation* //rs/tests/networking:nns_delegation_test
+
+rs/backup/*.rs //rs/tests/consensus/backup:backup_manager_upgrade_test
+               //rs/tests/consensus/backup:backup_manager_downgrade_test
```

Comment:
These tests are tagged as `experimental_system_test_colocation` so you need to use their `_colocate` variants:
```suggestion
rs/backup/*.rs //rs/tests/consensus/backup:backup_manager_upgrade_test_colocate
               //rs/tests/consensus/backup:backup_manager_downgrade_test_colocate
```

Reply:
Done: https://github.com/dfinity/ic/pull/6340